### ISSUE TITLE
Incorrect button layouts

### ISSUE THREAD

**Issue by mtrezza** [parse-community/parse-dashboard#1959]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description
It appears that https://github.com/parse-community/parse-dashboard/issues/1941 caused some minor layout issues for buttons, specifically margins and possibly font size.

cc @visualfanatic 

### Steps to reproduce
For example the data browser filter dialog:

Before #1941:
<img width="530" alt="Screen Shot 2021-12-12 at 16 02 44" src="https://user-images.githubusercontent.com/5673677/145717729-aa141dcf-543f-4bd4-90ab-cde534adc58f.png">

After #1941:
<img width="533" alt="Screen Shot 2021-12-12 at 16 02 07" src="https://user-images.githubusercontent.com/5673677/145717736-9f19253e-11c8-4e60-93b0-6e43f0621e85.png">

### Actual Outcome
n/a

### Expected Outcome
n/a

### Environment
- Parse Dashboard 4.0.0-alpha.5 

### Logs
n/a

**Comment by AshishBarvaliya** [comment]
@mtrezza Seems like the issue has been resolved.

**Comment by mtrezza** [comment]
Yes, this is quite an old one, thanks for pointing this out.
